WordPress/src/main/java/org/wordpress/android/ui/posts/PrepublishingViewModel.kt

```diff
@@ -1,13 +1,89 @@
 package org.wordpress.android.ui.posts
 
+import android.os.Bundle
+import android.os.Parcelable
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
+import kotlinx.android.parcel.Parcelize
+import org.wordpress.android.fluxc.model.SiteModel
+import org.wordpress.android.ui.posts.PrepublishingHomeItemUiState.ActionType
+import org.wordpress.android.ui.posts.PrepublishingScreen.HOME
+import org.wordpress.android.viewmodel.Event
 import javax.inject.Inject
 
+const val KEY_SCREEN_STATE = "key_screen_state"
+
+data class PrepublishingNavigationTarget(
```

Comment:
NitPick: We might want to consider moving even this class to the bottom of the file, wdyt?

Reply:
Makes sense, thanks. I enjoy your nitpicks a lot. Always share them 😅